### PR TITLE
feat(rust): update rust api examples with colored output

### PR DESCRIPTION
**Pull request by nazmulidris**
Update Rust API examples so that they have nice colorized output that is easy for developers to read and understand.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: examples/rust/get_started/examples/01-node.rs [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+
+/// Create and then immediately stop a node.
 #[ockam::node]
 async fn main(ctx: Context) -> Result<()> {
-    // Create a node with default implementations
+    println!("{}", HELP_TEXT.green());
+
+    print_title(vec!["Run a node & stop it right away"]);
+
+    // Create a node with default implementations.
```
Comment (mrinalwadhwa) [inline examples/rust/get_started/examples/01-node.rs]:
```suggestion
    // Create a node.
```

File: examples/rust/get_started/examples/01-node.rs [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+use colored::Colorize;
 use ockam::{node, Context, Result};
 
+#[rustfmt::skip]
+const HELP_TEXT: &str =r#"
+┌──────────────────────┐
+│  Node 1              │
+├──────────────────────┤
+│  ┌────────────────┐  │
+│  │ Address:       │  │
```
Comment (mrinalwadhwa) [inline examples/rust/get_started/examples/01-node.rs]:
```suggestion
│  │ Worker Address │  │
```

**Review by mrinalwadhwa — DISMISSED** [review]
@nazmulidris this looks great, left a couple of minor suggestions.

File: examples/rust/get_started/examples/01-node.rs (hunk 2 above)
Comment (nazmulidris) [reply examples/rust/get_started/examples/01-node.rs]: Done!

File: examples/rust/get_started/examples/01-node.rs (hunk 1 above)
Comment (nazmulidris) [reply examples/rust/get_started/examples/01-node.rs]: Done

**Comment by nazmulidris** [comment]
@mrinalwadhwa 
- Thank you for your suggestions 🙏🏽. I've incorporated them all. 
- I've also switched to using a different dependency for ansi 256 color output (that works on linux, macos, and windows & has a compatible license).

> Also, the `lint_docs_ockam_io_rust_examples` are failing in this PR because I've not merged this PR in the `ockam-documentation` repo yet https://github.com/build-trust/ockam-documentation/pull/56

**Review by nazmulidris — COMMENTED** [review]
@mrinalwadhwa I've made all the changes you requested 👍🏽

**Review by mrinalwadhwa — APPROVED** [review]

**Pull request merged**